### PR TITLE
Add initial GitHub issue templates

### PR DESCRIPTION
**Pull request by seanmalloy**
Initially users can open a bug report, feature request, or misc request.
Bug reports and feature requests will have the "kind" label
automatically added.

### CODE REVIEW

**Comment by seanmalloy** [comment]
/hold

I want to create a dummy repo with these files to test it out before merging.

**Comment by seanmalloy** [comment]
/hold cancel
/assign @damemi @ingvagabund 

I tried out the issue templates using repo, https://github.com/seanmalloy/test-repo. Feel free to open an issue in that repo if you want to try it out.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+descheduler version:
+
+
+**Does this issue reproduce with the latest release?**
+
+
+**Which descheduler CLI options are you using?**
+
+
+**Please provide a copy of your deschedler policy config file**
```
Comment (ingvagabund) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: s/deschedler/descheduler

Comment (seanmalloy) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed in f834581a8ee7c90063177edae66bdd5115246cab

**Comment by seanmalloy** [comment]
@ingvagabund I fixed the spelling error. Let me know if you see any other changes that are needed.

**Comment by ingvagabund** [comment]
> @ingvagabund I fixed the spelling error. Let me know if you see any other changes that are needed.
Nope, all set!!!

**Pull request merged**